Data/JobContext.cs

```diff
@@ -0,0 +1,26 @@
+﻿using API.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace API.Data
+{
+    public class JobContext : DbContext
```

Comment:
Det vil nok være en god ide at ændre navet på JobContext til AutismeHubContext for at det giver bedre mening på den lange bane. Da context classen ikke kun indeholder data omkring jobs, men også mange andre elementer

Reply:
Vi venter lidt med det så det ikke giver nogen merge conflicts